src/routes/routes.tsx

```diff
@@ -66,15 +66,15 @@ const LoggedInRoutes = () => {
       <Route path="circles/:circleId" element={<OrgRouteHandler />}>
         <Route path="activity" element={<CircleActivityPage />} />
         <Route path="members" element={<MembersPage />} />
+        <Route path="map" element={<MapRouteHandler />}>
+          <Route path="" element={<LazyAssetMapPage />} />
```

Comment:
is this path="" ? i'm not familiar with existing routes, just wanted to bring attention to making sure this was intended.

Reply:
yep, that's intentional. it means that once the parent route "map" is matched, this route should be executed for all nested routes. in this case it's just a way of running MapRouteHandler with LazyAssetMapPage nested inside it.